cypress/support/index.d.ts

```diff
@@ -1,9 +1 @@
 import '@mate-academy/cypress-tools/support';
-
```

Comment:
why did you delete it?

Reply:
I dont't know why this file was chanched((